packages/kbn-i18n/babel_config.js

```diff
@@ -17,18 +17,34 @@
  * under the License.
  */
 
-import * as i18n from '../core';
-
-export function i18nProvider() {
-  this.addMessages = i18n.addMessages;
-  this.getMessages = i18n.getMessages;
-  this.setLocale = i18n.setLocale;
-  this.getLocale = i18n.getLocale;
-  this.setDefaultLocale = i18n.setDefaultLocale;
-  this.getDefaultLocale = i18n.getDefaultLocale;
-  this.setFormats = i18n.setFormats;
-  this.getFormats = i18n.getFormats;
-  this.getRegisteredLocales = i18n.getRegisteredLocales;
-  this.init = i18n.init;
-  this.$get = () => i18n.translate;
-}
+module.exports = {
+  plugins: ['@babel/plugin-proposal-class-properties', '@babel/plugin-proposal-object-rest-spread'],
+  presets: ['@babel/preset-react', '@babel/typescript'],
+  env: {
+    web: {
+      presets: [
+        [
+          '@babel/env',
+          {
+            targets: {
+              browsers: ['last 2 versions', '> 5%', 'Safari 7'],
```

Comment:
__question:__ is there way we can reuse `packages/kbn-babel-preset` here? I'm really concerned that we may diverge with Kibana at some point. If it's not possible right now, please leave a proper comment here explaining why we're doing what we're doing and when we can leverage common preset.